From: Jan Engelhardt <jengelh@inai.de>
Date: 2015-12-08 13:49:00.250213202 +0100

dxflib by default does not build a shared library, and does not
track its ABI either. So we have to do that.

1: dxflib-3.7.5
suse2: dxflib-3.12.2: class DL_DimensionData gained new member

---
 dxflib.pro |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: dxflib-3.17.0-src/dxflib.pro
===================================================================
--- dxflib-3.17.0-src.orig/dxflib.pro
+++ dxflib-3.17.0-src/dxflib.pro
@@ -23,7 +23,7 @@ SOURCES = \
     src/dl_dxf.cpp \
     src/dl_writer_ascii.cpp
 
-TARGET = dxflib
+TARGET = dxflib-3.17.0
 TEMPLATE = lib
-CONFIG += staticlib
+CONFIG += sharedlib
 DEFINES += DXFLIB_LIBRARY
